resolwe/permissions/shortcuts.py

```diff
@@ -194,3 +198,165 @@ def format_permissions(perms):
         })
 
     return perms_list
+
+
+# based on guardian.shortcuts.get_objects_for_user
+def get_objects_for_user(user, perms, klass=None, use_groups=True, any_perm=False,
+                         with_superuser=True, accept_global_perms=True, perms_filter='pk__in'):
```

Comment:
Are just the lines that include `perms_filter` are different from the original implementation in REST framework?